src/components/ScreenMain/Footer/index.tsx

```diff
@@ -15,28 +15,31 @@ const FooterContainer = styled.div`
   display: grid;
   grid-template-columns: 1fr 2fr 1fr;
   width: 100%;
-  background-color: white;
   padding: 4px;
 `;
 
 const CharacterSection = styled.div`
   display: flex;
-  flex-flow: row wrap;
+  flex-flow: row no-wrap;
   justify-content: center;
   padding: 8px;
   gap: 16px;
   align-items: center;
+  font-size: 20px;
+  color: white;
 `;
 
 const StatSection = styled.div`
   display: flex;
-  flex-flow: row wrap;
+  flex-flow: row no-wrap;
   justify-content: center;
   gap: 8px;
+  align-items: center;
 `;
 
 const ToggleListSection = styled.div`
-  padding: 8px 8px 8px 0;
+  // padding: 8px 0 8px 8px;
```

Comment:
Suggestion: Commented out line can be deleted

Reply:
what kind of dumbo tries to commit commented out code?